Tests/scripts/destroy_instances.py

```diff
@@ -2,81 +2,126 @@
 import json
 import logging
 import os
+import sys
+from datetime import datetime
 from pathlib import Path
-import subprocess
-from typing import Optional
+
+import humanize
+import urllib3
 from demisto_sdk.commands.test_content.constants import SSH_USER
+from paramiko import SSHClient, SSHException
+from scp import SCPClient, SCPException
+
 from Tests.scripts.utils.log_util import install_logging
 
-# Disable insecure warnings
-import urllib3
-urllib3.disable_warnings()
+urllib3.disable_warnings()  # Disable insecure warnings
+
+DEFAULT_TTL = "300"
+SERVER_LOG_DIRECTORY = "/var/log/demisto"
+SERVER_LOG_FILE_PATH = f"{SERVER_LOG_DIRECTORY}/server.log"
 
 
 def options_handler():
-    parser = argparse.ArgumentParser(description='Utility for instantiating and testing integration instances')
-    parser.add_argument('--aritfacts-dir', help='Path to the artifacts directory', required=True)
+    parser = argparse.ArgumentParser(description='Utility for destroying integration test instances')
+    parser.add_argument('--artifacts-dir', help='Path to the artifacts directory', required=True)
     parser.add_argument('--instance-role', help='The instance role', required=True)
     parser.add_argument('--env-file', help='The env_results.json file')
     return parser.parse_args()
 
 
-def chmod_logs(server_ip: str):
+def chmod_logs(ssh: SSHClient, server_ip: str) -> bool:
     try:
-        logging.debug(f'Changing permissions of folder /var/log/demisto on server {server_ip}')
-        subprocess.check_output(f'ssh {SSH_USER}@{server_ip} "sudo chmod -R 755 /var/log/demisto"', shell=True)
+        logging.info(f'Changing permissions of folder {SERVER_LOG_DIRECTORY} on server {server_ip}')
+        _, stdout, stderr = ssh.exec_command(f"sudo chmod -R 755 {SERVER_LOG_DIRECTORY}")
+        # Wait for the command to exit.
+        exit_code = stdout.channel.recv_exit_status()
+        stderr_lines = stderr.readlines()
+        logging.info(f"Running chmod finished - stdout:{stdout.readlines()}, stderr:{stderr_lines}, exit code:{exit_code}")
```

Comment:
Aren't there multiple lines in the outputs? So maybe this is nicer?
```suggestion
        logging.info(f"Running chmod finished.\nstdout:\n{stdout.readlines()}\nstderr:\n{stderr_lines}\nexit code: exit_code}")
```

Reply:
will fix on my side